CHANGES.md

```diff
@@ -1,4 +1,189 @@
+## stable-2.3.0
+
+### Announcing Linkerd 2.3 🎈
+
+This stable release introduces a new TLS Identity system into the default
+Linkerd installation, replacing `tls=optional` and the `linkerd-ca` controller.
+Now, proxies generate ephemeral private keys into a tmpfs directory and
+dynamically refresh certificates, authenticated by Kubernetes ServiceAccount
+tokens, via the newly-introduced Identity controller.
+
+Now, all meshed HTTP communication is private and authenticated by default.
+
+In the CLI, this release includes 2 new commands:
+* `linkerd metrics` fetches metrics directly from a Linkerd proxy
+* `linkerd upgrade` upgrades your existing Linkerd installation
```

Comment:
This command won't work for upgrading from `stable-2.2.1` to `stable-2.3.0`, right? For users upgrading from 2.2, we're still recommending that they use `linkerd install`? Maybe we could note that here and link to an upgrade guide that explains in more detail?

Reply:
On second thought, since you have a section below for "Upgrade notes", I'd recommend removing this paragraph entirely. I'm not sure it's worth calling out the two new CLI commands separately, especially since one of them is unusable for stable installs.